src/main/java/net/mcreator/element/parts/gui/GUIComponent.java

```diff
@@ -41,6 +41,7 @@
 	public int x;
 	public int y;
 
+	public transient boolean locked = false;

```

Comment:
Maybe it would make sense to actually store this parameter?

Reply:
I didn't know which one was better.
Now I think it makes more sense to store it, since it is time-consuming to lock the same components every time you reopen the mod element.